src/main/app/initialize.ts

```diff
@@ -430,6 +434,7 @@ async function initializeAfterAppReady() {
     }
 
     handleUpdateMenuEvent();
+    DeveloperMode.on(DEVELOPER_MODE_UPDATED, handleUpdateMenuEvent);
```

Comment:
Shouldn't we remove line 436?

Reply:
The event only emits when we toggle a developer tool, we still need to create the menu in the first place which is what line 436 does.